pkg/kube/pod_controller.go

```diff
@@ -0,0 +1,212 @@
+package kube
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
```

Note, you have it already imported as metav1.

Reply:
Thanks for pointing, removed.